PARAM_DIFFERENCES.md

```diff
@@ -0,0 +1,57 @@
+## Parameter Differences
+
```

Comment:
please add a brief explanatory note: "This is a comparison of parameters for python versions of this repository htm.core (link) and Numenta's NuPIC (link). Along with the API_CHANGELOG (link) this serves as a guide when migrating your code or comparing the frameworks. "

Reply:
Done 👍